Makefile

```diff
@@ -25,7 +25,8 @@ ppx_inline_test \
 ppxlib \
 ctypes \
 "utop>=2.6.0" \
-"melange"
+"melange" \
+"ocaml-index"
```

Comment:
it's best if `ocaml-index` is not a test-dep. you can write a fake binary that still allows you to test the behavior (I can help with that)

Reply:
Could you point me to such "fake binary" in the tree that I could take inspiration from ?

Reply:
`test/blackbox-tests/utils/refmt.ml`

Reply:
I tried to do something similar but I had to hack some workaround because the binary doesn't have a valid module name (`ocaml-index`). Is there a better way to do it ?

It's in that commit: https://github.com/ocaml/dune/pull/10422/commits/cc7c74e29f61480b5d93cb782b8782027c49bd77